templates/app-nolayers/aspnet-core/MyCompanyName.MyProjectName.Mvc.Mongo/MyCompanyName.MyProjectName.Mvc.Mongo.csproj

```diff
@@ -18,7 +18,7 @@
     <ProjectReference Include="..\..\..\..\framework\src\Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared\Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared.csproj" />
     <ProjectReference Include="..\..\..\..\framework\src\Volo.Abp.AutoMapper\Volo.Abp.AutoMapper.csproj" />
     <!-- </TEMPLATE-REMOVE> -->
-    <PackageReference Include="Volo.Abp.AspNetCore.Mvc.UI.Theme.LeptonXLite" Version="3.2.0" />
+    <ProjectReference Include="..\..\..\..\..\..\lepton\aspnet-core\abp\src\Volo.Abp.AspNetCore.Mvc.UI.Theme.LeptonXLite\Volo.Abp.AspNetCore.Mvc.UI.Theme.LeptonXLite.csproj" />
```

Comment:
This might not be replaced by the ABP CLI? I guess this is added for testing purposes right? @maliming

Reply:
You are right, this is the test code. I have reverted the code.